pages/linux/mmcli.md

```diff
@@ -0,0 +1,12 @@
+# mmcli
+
+> Control and monitor the ModemManager.
+> More information: <https://www.freedesktop.org/software/ModemManager/man/1.0.0/mmcli.8.html>.
+
+- List SMS messages available on the modem:
+
+`sudo mmcli -m {{modem}} --messaging-list-sms`
```

Comment:
We prefer long arguments when there are available, checking the man page we can use:
```suggestion
`sudo mmcli --modem={{modem}} --messaging-list-sms`
```